jenkins/gradle/gradle-check.jenkinsfile

```diff
@@ -129,6 +129,9 @@ pipeline {
                 }
             }
             post() {
+                failure {
+                    archiveArtifacts artifacts: 'search/java_pid*.hprof'
```

Comment:
@peterzhuamazon could we change it please to  `archiveArtifacts artifacts: '**/build/heapdump/*.hprof', allowEmptyArchive: true`? thank you

Reply:
You got it.